### ISSUE TITLE
Retrn Error during installing CouchDB by generate_box.sh

### ISSUE THREAD

**Issue by KoukiHama** [sw360/sw360vagrant#39]
> ==> sw360-focal: Notice: /Stage[main]/Sw360base/Exec[install_thrift]/returns: Reading package lists...
==> sw360-focal: Notice: /Stage[main]/Sw360base/Exec[install_thrift]/returns: E: Failed to fetch https://apache.bintray.com/couchdb-deb/dists/focal/InRelease  403  Forbidden [IP: 34.215.50.10 443]
==> sw360-focal: Notice: /Stage[main]/Sw360base/Exec[install_thrift]/returns: E: The repository 'https://apache.bintray.com/couchdb-deb focal InRelease' is not signed.

This is same with this one.
https://github.com/eclipse/sw360/issues/1331

Using https://apache.jfrog.io/artifactory/couchdb-deb is nesesaery.
because, https://apache.bintray.com/couchdb-deb does not work well now.

**Comment by drgsr** [comment]
Hi - I stubled on right the very same trap...
Obviously, the repo  (apache.bintray) of couchdb got moved to jfrog ...
See e.g.
https://docs.couchdb.org/en/stable/install/unix.html#installation-using-the-apache-couchdb-convenience-binary-packages
Quote from that page...
```
sudo apt update && sudo apt install -y curl apt-transport-https gnupg
curl https://couchdb.apache.org/repo/keys.asc | gpg --dearmor | sudo tee /usr/share/keyrings/couchdb-archive-keyring.gpg >/dev/null 2>&1
source /etc/os-release
echo "deb [signed-by=/usr/share/keyrings/couchdb-archive-keyring.gpg] https://apache.jfrog.io/artifactory/couchdb-deb/ ${VERSION_CODENAME} main" \
    | sudo tee /etc/apt/sources.list.d/couchdb.list >/dev/null
```
 To Fix the issue, one needs to edit the file   "puppet/manifests/sw360-base.pp" 
 Exchange the lines:
 `$puppet_does_not_allow_escaping='deb https://apache.bintray.com/couchdb-deb focal main'`
to become:
`  $puppet_does_not_allow_escaping='deb https://apache.jfrog.io/artifactory/couchdb-deb/  focal main'`
 AND 
  `command => "curl -L https://couchdb.apache.org/repo/bintray-pubkey.asc | sudo apt-key add -",`
to become
   `command => "curl -L https://couchdb.apache.org/repo/keys.asc | sudo apt-key add -",`

**Comment by mcjaeger** [comment]
going to work on it.

**Comment by KoukiHama** [comment]
This pull request try to fix this issue.
https://github.com/sw360/sw360vagrant/pull/40

**Comment by mcjaeger** [comment]
yes, I am testing it right now

**Comment by KoukiHama** [comment]
Solved by latest commit 2060f96
